lib/plugin.py

```diff
@@ -66,10 +62,7 @@ def do_authenticate(self):
             self.logger.error("Network error while talking to RADIUS server: %s", ex)
             return AAResponse.deny(deny_reason=DenyReasons().communication_error)
         except Exception as ex:
-            self.logger.error(
-                "An exception of type %s occured. Arguments:\n%s", type(ex).__name__, ex.args,
-            )
-            self.logger.debug("Exception details follow.", exc_info=ex)
+            self.logger.error("An exception of type %s occurred:\n%s", type(ex).__name__, ex)
```

Comment:
The new line won't make this log look strange in the SPS logs?

Reply:
After IRL discussion we will change this to:
Unknown exception: %s